src/Components/Replication/ReplicationSettingsForm.tsx

```diff
@@ -78,9 +91,15 @@ export default class ReplicationSettingsFormReplication extends React.Component<
     entries: [],
     include: {},
     exclude: {},
+    when: "",
     dataFetched: false,
+    lastRefresh: Date.now(),
```

Comment:
I believe we don't need any timers or asynchronous interactions with the form or the databases.